guide/getting-started/12-technology-primer.md

```diff
@@ -0,0 +1,30 @@
+---
+
+# Bitcoin technology primer
+
+<!--
+
+It can take many years to understand bitcoin in every detail, but luckily you don’t need to know everything to design great products. Let’s take a look at the most important aspects of the technology behind bitcoin, with an eye towards the basics needed to design bitcoin products.
+
+The most fundamental read is the Bitcoin White Paper. Published on October 31, 2008 by the mysterious Satoshi Nakamoto, it layous the fundamental system design in only 9 short pages.
+
+The white paper describes Bitcoin as electronic peer-to-peer cash. Just like cash, users are fully in control of their Bitcoin, and also like cash, they can send Bitcoin to anyone they want, without any middlemen. This is achieved through a network of thousands of computers that are in constant communication with each other to pass back and forth transactions. These so-called nodes ensure that users can only spend Bitcoin they own, and that they are only spent once. Anyone with a computer can run a node, and many people do, as it provides them with extra certainty that their Bitcoin are secure.
+
+So how do you actually own Bitcoin? In simple terms, owning 1 Bitcoin means that you have control over a Bitcoin address that somebody else sent 1 Bitcoin to.
+
+Addresses can easily be generated on a smartphone or computer via a Bitcoin wallet application. Every address comes with a matching private key, which allows you to send the Bitcoin associated with the address. As the name implies, it is very important to keep those private keys secure, as anyone can use them to control your hard-earned Bitcoin.
+
+While we typically use one credit card number or have one bank account number in traditional banking, addresses in Bitcoin offer a bit more flexibility. While it is not required, it is recommended that a new address is generated for every transaction. Since all Bitcoin transactions are public, reusing the same address makes it easy for others to analyze your financial habits.
+That’s where wallets come in. A Bitcoin wallet is typically based on a so-called recovery phrase, which is similar to private keys of addresses mentioned above. From a recovery phrase, wallets can generate many unique addresses.
+
+This mechanism of generating a piece of information from another one, where one can be shared and the other one is kept private is known as public-key cryptography. While there are many other cryptographic techniques used in Bitcoin, this is one that you should understand well.
+
+So you may ask how Bitcoin are created? This involves a process we know as mining. Miners are computers that help process transactions. For this effort, which requires huge amounts of computing power, they are rewarded with freshly minted Bitcoin. At the moment, 6.25 new Bitcoin are created around every 10 minutes, which is an annual inflation rate of 1.86%. The creation of Bitcoin is tightly controlled via code because money only has value if it is scarce.
```

Comment:
I would say one of the properties of good money is scarcity but not the only thing that gives it value. e.g durability, portability, fungible, divisible etc

Reply:
That leads us back to the other point you made about bitcoin being (or not being) a global currency, and how that is defined. Maybe we do need a section on that, or link to a good resource that explains it ([example](https://www.investopedia.com/ask/answers/100314/why-do-bitcoins-have-value.asp)). We might end up having to refer to these properties in a lot of other sections, too.